movies-manager/webpack.config.prod.js

```diff
@@ -0,0 +1,21 @@
+const path = require('path');
+const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
+const buildFolderName = 'build';
+
+module.exports = {
+  output: {
+    path: path.join(__dirname, buildFolderName),
+    filename: 'main.[hash].js',
+    chunkFilename: '[id].[hash].js',
+  },
+  devServer: {
```

Comment:
You don't this config anymore

Reply:
Also noticed that config.prod.js and config.dev.js have simmilar lines of code. Maybe you should combine this files in a one?

Reply:
for sure, done